person.rb

```diff
@@ -22,6 +23,11 @@ def correct_name
     name
   end
 
+  def add_rental(book, date)
+    rental = Rental.new(date, self, book)
+    rentals.push(rental)
+  end
+
   private
 
   def of_age?
```

Comment:
- [ ] [optional] @BilalLiaquat7 Kindly consider removing the code to test if the decoration classes are working, this making your interface a bit confusing 
![image](https://github.com/BilalLiaquat7/school_library/assets/61408860/34806bad-419d-40c8-81e8-aa1c8235daf0)